9armbot/9armbot.js

```diff
@@ -369,22 +377,40 @@ client.on('message', (channel, tags, message, self) => {
         }
     }
 
-    if (message == '!income' && player.isAdmin(tags.username)){
+    if (message == '!income' && player.isAdmin(tags.username)) {
         client.say(channel, `Payout Total: ${sessionPayout} armcoin. Gacha Total = ${sessionIncome} Net: ${sessionIncome - sessionPayout}`);
     }
 
-    if (message == '!save' && player.isAdmin(tags.username)){
+    if (message == '!save' && player.isAdmin(tags.username)) {
         saveBotData();
         player.saveData();
     }
 
-    if (message == '!load' && player.isAdmin(tags.username)){
+    if (message == '!load' && player.isAdmin(tags.username)) {
         restoreBotData();
     }
 });
 
-function subscriptionPayout (channel, username) {
-    botInfo.critRate+=0.1;
+async function onKick(channel, user) {
+    kick_command = user;
+    for (let i in kick_command) {
+        let kick_user = kick_command[i]
+        // username cannot start with an underscore _
+        let kick = kick_user.match(/^@?[a-zA-Z0-9][\w]{2,24}$/);
+        if (kick) {
+            let user = {
+                username: kick.input,
+                isMod: false
+            };
+            // client.timeout(channel, kick.input, 10)
+            await sleep(700);
```

Comment:
No need for this timeout as we wont kick that many people in a row. Thanos needs this because of rate limit about 100 kick/min.